HCCGo/app/js/clusterLandingCtrl.js

```diff
@@ -99,6 +100,44 @@ clusterLandingModule.controller('clusterLandingCtrl', ['$scope', '$log', '$timeo
     $event.stopPropagation();
   }
 
+  $scope.cancelModal = function(job, $event) {
+    $event.stopPropagation();
+    $scope.cancelJob = job;
+    $("#cancel").modal("show");
+  }
+
+  $scope.cancelRunningJob = function(job, $event) {
+    job.running = false;
+    job.idle = false;
+    job.cancelled = true;
```

Comment:
Ah, I missed these changes.  This is what I was talking about.  This should work.